src/components/SlateEditor/plugins/conceptList/ConceptList.tsx

```diff
@@ -56,10 +56,10 @@ const ConceptList = ({ element, editor, attributes, children }: Props) => {
 
   const conceptsQuery = useSearchConcepts(
     {
-      subjects: element.data?.subjectId,
-      tags: element.data?.tag,
+      ...(element.data?.subjectId ? { subjects: [element.data.subjectId] } : {}),
+      ...(element.data?.tag ? { tags: [element.data.tag] } : {}),
```

Comment:
Kunne vi kanskje laget en helper for dette? Syntaxen er såpass kjip at jeg kan se for meg at den er litt bug-prone.

Reply:
Lagde et forslag på det nå, vet ikke om jeg syntes det ble 10/10, men du kan se om du har noen innspill!